src/app/ReadPrepareParams.h

```diff
@@ -47,7 +47,7 @@ struct ReadPrepareParams
     uint16_t mMaxIntervalCeilingSeconds = 0;
     bool mKeepSubscriptions             = false;
     bool mIsFabricFiltered              = true;
-    bool mIsPeerLIT                     = false;
+    bool mPeerICD                       = false;
```

Comment:
Needs documentation, or better naming, or both, if it's meant to be "supports check-in protocol".

Reply:
this change is removed, and unnecessary.